### PR TITLE
[21446] Fast DDS Monitor refactor: Fast DDS v3 compatibility

### PR DESCRIPTION
**Pull request by Carlosespicur**
Add minor changes for Monitor compilation with Fast DDS v3. Fast DDS Statistics Backend migration to Fast DDS v3 (https://github.com/eProsima/Fast-DDS-statistics-backend/pull/239) required.  

**Related PRs**
- https://github.com/eProsima/Fast-DDS-statistics-backend/pull/239

### CODE REVIEW

File: docs/rst/notes/notes.rst
Comment (JesusPoderoso) [inline docs/rst/notes/notes.rst:1]: I think these changes are unnecessary, as long as the product name is displayed in the first column of the table (see [here](https://fast-dds-monitor.readthedocs.io/en/latest/rst/notes/notes.html)).

File: docs/rst/notes/previous_versions/v2.0.0.rst
Comment (JesusPoderoso) [inline docs/rst/notes/previous_versions/v2.0.0.rst:1]: Same comment as the previous suggestion.
I would keep the `.. _version_v2.0.0:` link but restore the remaining changes

File: docs/rst/ros/galactic/galactic.rst
Comment (JesusPoderoso) [inline docs/rst/ros/galactic/galactic.rst:1]: This document references a previous Fast DDS version (2.9.x) associated with ROS 2 Galactic, so it must not be updated. Please remove the changes

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]

File: docs/rst/notes/notes.rst
Comment (Carlosespicur) [reply docs/rst/notes/notes.rst:1]: This changes have been made to avoid a cross reference with the previous versions

**Review by JesusPoderoso — DISMISSED** [review]
LGTM with green CI
Lets wait the backend PR to get merged to trigger it

**Pull request merged**